04-gas/gas-section-two/gas-section-two.html

```diff
@@ -16,20 +16,27 @@
 
       <section class="gas-section-two-subcontainer">
 
-        <div class="gas-sectin-two-title-container">
-          <h2 class="gas-section-two-subtitle">2</h2>
-          <h1 class="gas-section-two-title">
-            Only turn on the heating when you need it
-          </h1>
+        <div class="gas-section-two-copy-container">
+
+          <div class="gas-section-two-title-container">
+            <h1 class="gas-section-two-title">2</h1>
+            <h1 class="gas-section-two-title">
+              Only <span class="bold">turn on the heating</span> when you<span class="bold"> need it</span>
+            </h1>
+          </div>
+
+          <div class="gas-section-two-paragraph-container">
+          
```

Comment:
remove this space, we can group it by div, i think its enough spacing... i know i haven't been consistent with you on this... thats my fault

Reply:
no worries i will change